tests/fixtures/mapping-destinations.yml

```diff
@@ -303,3 +310,15 @@ destinations:
     scheduling:
       accept:
         - pulsar-canberra
+  clamped_destination:
```

Comment:
is this supposed to be in a `destinations` dict?

Reply:
Yes, it's nested under destinations.

Reply:
yep, I misread this.